crepecake/src/main/java/com/google/cloud/tools/crepecake/builder/Step.java

```diff
@@ -16,9 +16,19 @@
 
 package com.google.cloud.tools.crepecake.builder;
 
+import com.google.cloud.tools.crepecake.image.DuplicateLayerException;
+import com.google.cloud.tools.crepecake.image.LayerCountMismatchException;
+import com.google.cloud.tools.crepecake.image.LayerPropertyNotFoundException;
+import com.google.cloud.tools.crepecake.registry.RegistryAuthenticationFailedException;
+import com.google.cloud.tools.crepecake.registry.RegistryException;
+import java.io.IOException;
+
 /** A step in the builder process. Implementations must be thread-safe. */
 interface Step<T, R> {
 
   // TODO: Add more exceptions as needed.
-  R run(T input);
+  R run(T input)
```

Comment:
Since we doesn't seem like we're using Step in a List<Step> or anything, perhaps this interface doesn't make sense, especially since Run seems to inflating the exception thrown list based on each individual Step's impl.

Reply:
Removing this in favor of implementing `Callable` to fit with `ExecutorService#submit`.